src/main/java/frc/robot/DriveBase.java

```diff
@@ -16,6 +16,8 @@
     private Gyroscope gyro;
     private TalonSRX talLM, talLF, talRM, talRF, talC;
 
+    Joystick joyL, joyR;
```

Comment:
Don't use Joysticks in DriveBase! We are only using those in Robot.java, so please remove them from this class along with associated commands.